src/test/java/org/elasticsearch/test/rest/ElasticsearchRestTestCase.java

```diff
@@ -302,4 +321,86 @@ public void test() throws IOException {
             executableSection.execute(restTestExecutionContext);
         }
     }
+
+    // don't look any further: NO TOUCHY!
+    
+    public static class Rest0Tests extends ElasticsearchRestTestCase {
```

Comment:
@s1monw that line only turns on some validation on unique suite names I think. It doesn't solve the reporting problems that we'd have without this hack (using @ReplicateOnEachVM).